src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj

```diff
@@ -23,10 +23,6 @@
     <Compile Include="System.Runtime.WindowsRuntime.Manual.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ReferencePath Include="$(ExternalAPIsPath)\NetFX\Contracts\DesignTimeFacades\mscorlib.dll" />
```

Comment:
Interesting. I suspect the metadata reader doesn't need to go through mscorlib references for these primitive types. That is nice, I didn't realize that was the case. At any rate I added Microsoft.NETCore.Portable.Compatibility package reference to get the mscorlib dependency.